omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/Tag.java

```diff
@@ -54,4 +54,8 @@ public void setDescription(String description) {
 		this.description = description;
 	}
 	
+	@Override
+	public int compareTo(Tag tag) {
+		return name.compareToIgnoreCase(tag.getName());

```

Comment:
From [here](https://docs.oracle.com/javase/7/docs/api/java/lang/Comparable.html):

> Note that `null` is not an instance of any class, and `e.compareTo(null)` should throw a `NullPointerException`

This is what will happen, but I could make it explicit if you prefer?